README.md

```diff
@@ -4,33 +4,40 @@
 - [Licenses](#licenses)
 - [Changelog](#changelog)
 
+##What's included
+This repository contains the core styles used across all aspects of Fabric including icons, type, fonts, colors, the grid, etc. If you have any issues or pull requests related to the JS components, please do so in the [Office UI Fabric JS repository](https://github.com/OfficeDev/office-ui-fabric-js). For more information about the React components, check out the [Office UI Fabric React repository](https://github.com/OfficeDev/office-ui-fabric-react).
+
+For documentation on Fabric's last stable version that included components, check out the [Fabric 2.6.1 get started guide](https://github.com/OfficeDev/office-ui-fabric-core/blob/master/ghdocs/V2GETSTARTED.md).
+
 
 ##Get started
-For a quick start, reference the latest release of Fabric from a CDN or add a copy to your project. See [Get Started](http://dev.office.com/fabric/get-started) on the [Office UI Fabric site](http://dev.office.com/fabric) for full details.
+For a quick start, reference the latest release of Fabric from a CDN or add a copy to your project. See [get started](http://dev.office.com/fabric/get-started) on the [Office UI Fabric site](http://dev.office.com/fabric) for full details on the most common ways to get started with the core along with a description of what's included.
+
+If you'd like to get Fabric through a package manager such as Bower, npm, or NuGet, check out the [package manager docs](https://github.com/OfficeDev/office-ui-fabric-core/blob/master/ghdocs/PACKAGES.md).
 
-Want to customize Fabric for your project? See [Building Fabric](https://github.com/OfficeDev/Office-UI-Fabric/blob/master/ghdocs/BUILDING.md) to learn about the build process.
+Want to customize Fabric for your project? See [building Fabric](https://github.com/OfficeDev/Office-UI-Fabric/blob/master/ghdocs/BUILDING.md) to learn about the build process.
 
 
 ##Learn more
 
-New to Fabric? Start with [What is Office UI Fabric?](http://dev.office.com/fabric/what-is-fabric) to understand its purpose, and then explore the full set of [styles](http://dev.office.com/fabric/styles) and [components](http://dev.office.com/fabric/components) to get a feel for what you can build. Components come in two flavors provided by the Fabric team: [JavaScript (Office UI Fabric JS)](https://github.com/OfficeDev/office-ui-fabric-js) and [React (Office UI Fabric React)](https://github.com/OfficeDev/office-ui-fabric-react).
+New to Fabric? Start with [the project's site](http://dev.office.com/fabric) to understand its purpose, and then explore the full set of [styles](http://dev.office.com/fabric/styles) to get a feel for what you can build. Components are contained in separate projects and come in two flavors provided by the Fabric team: [JavaScript (Office UI Fabric JS)](https://github.com/OfficeDev/office-ui-fabric-js) and [React (Office UI Fabric React)](https://github.com/OfficeDev/office-ui-fabric-react).
```

Comment:
In this context, it **may** be helpful to specifically say something like "UI components" for the first instance instead of just "Components"--new readers may not know exactly what we mean (you could argue that "typography" is a "component" of Fabric).